Include/internal/pycore_gc.h

```diff
@@ -187,23 +201,16 @@ struct _gc_runtime_state {
     PyObject *garbage;
     /* a list of callbacks to be invoked when collection is performed */
     PyObject *callbacks;
-    /* This is the number of objects that survived the last full
-       collection. It approximates the number of long lived objects
-       tracked by the GC.
-
-       (by "full collection", we mean a collection of the oldest
-       generation). */
-    Py_ssize_t long_lived_total;
-    /* This is the number of objects that survived all "non-full"
-       collections, and are awaiting to undergo a full collection for
-       the first time. */
-    Py_ssize_t long_lived_pending;
+
+    Py_ssize_t incremental_gc_progress;
```

Comment:
It seems like this name is maybe not the best?  It doesn't really represent the progress of the gc, but rather the threshold until the next incremental gc.  Maybe "progress" here means progress until the next gc, but I think it's easily confused with progress through an incremental gc, which isn't really just a local thing.

Reply:
I'll rename it from "progress" to "work_done", since it is the negative of "work to do".